game-core/src/main/java/games/strategy/triplea/ai/pro/ProCombatMoveAi.java

```diff
@@ -1034,8 +1034,10 @@ private void determineUnitsToAttackWith(final List<ProTerritory> prioritizedTerr
     // Set enough land and sea units in territories to have at least a chance of winning
     for (final Unit unit : sortedUnitAttackOptions.keySet()) {
       final boolean isAirUnit = UnitAttachment.get(unit.getType()).getIsAir();
-      if (isAirUnit || addedUnits.contains(unit)) {
-        continue; // skip air units
+      final boolean isExpensiveLandUnit = Matches.unitIsLand().test(unit)
+          && ProData.unitValueMap.getInt(unit.getType()) > 2 * ProData.minCostPerHitPoint;
+      if (isAirUnit || isExpensiveLandUnit || addedUnits.contains(unit)) {
+        continue; // skip air and expensive units
```

Comment:
Want to use cheap land units to populate attacks in territories first as fodder. Most maps this doesn't matter since all land units are fairly cheap but some of the advanced maps have expensive land units.